Automattic/jetpack
# SSO: Update user-new.php user email and invitation logic

Fixes #

## Proposed changes:

These changes:
* Refactor the `jetpack-sso-admin-create-user` CSS and JS to use the preferred Jetpack script registering and enqueuing functions.
* Unchecks the `Invite user` checkbox on page load if the WooCommerce plugin is activated.
* Add logic to disable the `Send User Notification` checkbox if the `Invite User` checkbox is checked.



### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Sync to Atomic site
* Make sure the WooCommerce plugin is deactivated if you have it installed.
* Navigate to `/wp-admin/user-new.php`
* The 'Invite users` checkbox should be enabled and `Send User Notification` checkbox should be disabled on page load.
* Verify that checking the `Invite users` checkbox disables the `Send User Notification` checkbox.
* Create a new user and verify that the `Invite user` checkbox is working correctly (checked and unchecked). This should send a user invitation email to join your site if it's checked.
* Create a new user and verify that the `Send user Notification` checkbox is working correctly (checked and unchecked while enabled).
This should send a "login details" email if it's checked.
* Install the WooCommerce plugin and activate it.
* Navigate back to `/wp-admin/user-new.php`.
* Verify the `Invite user` checkbox is unchecked on page load.
* Verify the `Send User Notification` checkbox is checked and enabled on page load.
* Retest checkbox behavior by creating new users while WooCommerce is activated.
